common/views/components/Quote/README.md

```diff
@@ -2,4 +2,31 @@
 
 Display a block or pull quote.
 
+## Usage
+
+**Block quote style**
+- Used as part of a narrative flow
+- Requires a citation
```

Comment:
I’m not sure this is always true (if e.g. the source is clear from the context).

Reply:
You're right. I'll change it.